velox/exec/Spill.h

```diff
@@ -429,26 +429,27 @@ class FileSpillBatchStream : public BatchStream {
 /// start bit offset is used to calculate the partition number of spill data. It
 /// is required for the recursive spilling handling as we advance the start bit
 /// offset when we go to the next level of recursive spilling.
-struct SpillPartitionId {
+///
+/// NOTE: multiple of shards created from the same SpillPartition by split()
```

Comment:
typo: "multiple of shards" -> "Multiple shards"

You are using a term "shard" here and "split" elsewhere. Should we use "split" here or start using "shard" everywhere? Shard might be better because we already use "split" for table scan splits and because split and spill words are very similar and hard to read in the same sentence.

Reply:
Updated.